docs/configuration/overview/runtime.rst

```diff
@@ -33,7 +33,51 @@ override_subdirectory
   specified and the directory exists, configuration values within this directory will override those
   found in the primary subdirectory. This is useful when Envoy is deployed across many different
   types of servers. Sometimes it is useful to have a per service cluster directory for runtime
-  configuration.
+  configuration. See below for exactly how the override directory is used.
+
+File system layout
+------------------
+
+Various sections of the configuration guide describe the runtime settings that are available.
+For example, :ref:`here <config_cluster_manager_cluster_runtime>` are the runtime settings for
+upstream clusters.
+
+Assume the following configuration settings for runtime:
```

Comment:
Assume that the folder /etc/config/v1/envoy points to the actual filesystem where global envoy runtime configurations are stored. The following would be a typical configuration setting for the runtime:
...

where, /srv/runtime/current would be a symlink to /etc/config/v1.